### PR TITLE
KPMP-1491: Check user

### PR DESCRIPTION
**Pull request by rlreamy**

### CODE REVIEW

File: src/components/Packages/PackagePanelStateText.js [hunk 1]
```diff
@@ -52,6 +53,20 @@ class PackagePanelStateText extends Component {
         });
     }
 
+    shouldShowIcon() {
+    	console.log(this.props.currentUser);
+    	console.log(this.props.packageSubmitter);
+    	
+    	if (this.props.currentUser.shibId === this.props.packageSubmitter.shibId) {
+    		console.log("shibIds equal");
```
Comment (rossmithumms) [inline src/components/Packages/PackagePanelStateText.js]: Please remove this log.

File: src/components/Packages/PackagePanelStateText.js [hunk 2]
```diff
@@ -52,6 +53,20 @@ class PackagePanelStateText extends Component {
...
+    		let panelConfig = PANEL_CONFIGS[this.props.panelState.state];
+    		return panelConfig.icon && this.props.handleStateInfoClick;
+    	} else {
+    		console.log("Shibids not equal");
```
Comment (rossmithumms) [inline src/components/Packages/PackagePanelStateText.js]: Please remove this log.

File: src/components/Packages/PackagePanelStateText.js [hunk 3]
```diff
@@ -52,6 +53,20 @@ class PackagePanelStateText extends Component {
         });
     }
 
+    shouldShowIcon() {
```
Comment (rossmithumms) [inline src/components/Packages/PackagePanelStateText.js]: Please give this a more semantic name (something like `isInteractive`).

Comment (rlreamy) [reply src/components/Packages/PackagePanelStateText.js]: I think this name is actually very descriptive.  We are not showing an icon at all if the user does not own this package...it doesn't make it not interactive, just hides the icon.

**Pull request merged**